### ISSUE TITLE
Example usage or rpcapd implementation of this protocol?

### ISSUE THREAD

**Issue by daluu** [kor44/rpcap#1]
Hi, this is a nice project. I assume the protocol implementation can be used to implement a rpcapd server/daemon or a rpcap client?

I was just wondering beyond the unit tests, is there any example of how this is made use of in an actual server or client using rpcap? Or a reference implementation of sorts somewhere?

Or is this just a barebones spec implementation for Proof of Concept? Or the spec was made open source but not the reference implementation that uses it?

**Comment by kor44** [comment]
Hi
In fact I tried to implement RPCAP protocol of libpcap library to understand how it works and had plans to make rpcapd analog but due to lack of time, I abandoned this idea. If you are interesting I can try to make simple example

**Comment by daluu** [comment]
I'm trying to understand the RPCAP protocol as well and any examples you have to actually using/implementing the protocol would be much appreciated!

It's too bad there doesn't seem to be much of a community around RPCAP development outside of libpcap (and Wireshark on the client side). And too bad the RPCAP protocol has never been documented like a spec like HTTP, where you have to analyze the source code and libpcap APIs for details.

**Comment by kor44** [comment]
Yes you are right. There are no information about RPCAP internals. I have read source code to understand how it works.
I will try to add some information here.

**Comment by kor44** [comment]
I have added information on wiki of project. Quality is not good but it's better then nothing.
May be it will be usefull
